src/content/staking/pools/index.md

```diff
@@ -72,6 +72,14 @@ Currently, withdrawing funds from a validator on the Beacon Chain is not possibl
 Alternatively, pools that utilize an ERC-20 liquidity token allow users to trade this token in the open market, effectively allowing you to "withdraw" without actually removing ETH from the Beacon Chain.
 </ExpandableCard>
 
+<ExpandableCard title="I am a pooled staker, do I need to do anything for The Merge?">
+ETH/token holders, pooled/SaaS stakers and general Ethereum users <strong>do not need to do anything to prepare for The Merge</strong>.
```

Comment:
I'd prefer to answer the question we're being asking here instead of trying to generalise advice. This page is focused on pooled stakers so we should focus on them.

```suggestion
Pooled stakers <strong>do not need to do anything to prepare for The Merge</strong>.
```